modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -7,8 +7,19 @@ import {Page, mergeTests} from '@playwright/test';
 
 import {backendPageTest} from './backendPageTest';
 
+interface ScopedFeatureFlagValue {
+	enabled: boolean;
+	system: boolean;
+}
+
 export interface FeatureFlagsOptions {
-	[key: string]: boolean;
+	[key: string]: boolean | ScopedFeatureFlagValue;
+}
+
+export interface FeatureFlagBody {
```

Comment:
Same here, I think we should be adding companyId as an optional attribute on the FeatureFlag interface.